README.md

```diff
@@ -44,57 +44,57 @@ To build all artifacts of this repository use the command `./gradlew packages`.
+    3. Persist the model with as much metadata as a graph in a `Neo4j` database.
+The commits of the same project have all their models stored in one database. That is, by the end of this step, we have a `Neo4J` database per project.
+4. We run `SmellTracker.jar` on each project database to fill a `PostgreSQL` database containing valuable data fo all applications.
+The `SmellTracker` process is detailed in [this document](./SmellTracker/docs/process.pdf), and can be resumed as follows:
+    1. Extract the commits data and order from the *Git* repository and the `Neo4j` database.
+    2. Retrieve the branch data and order from the *Git* repository. This step assures the precision of our smell history tracking.
+    3. Detect code smells by launching queries defined in `SmellDetector` on the `Neo4J` database.
+    4. Based on the extracted commits order and the detected code smells, track the history of each code smell instance and store it in the `PostgreSQL` database.
 
 ## Integrate new smells
 
-### Add a new smell query
+### Add a new code smell query
 
-The easiest way to integrate a new smell in this process would be to create a new definition in `SmellDetector`.
+The easiest way to integrate a new code smell in this process would be to create a new definition in `SmellDetector`.
 This means writing a `Neo4j` query relying on the metadata persisted by `SmellDetector`.
-It may require to add some new metadata in the persisted model, but won't be much of a hassle to integrate.
+It may require to add some new metadata in the persisted model, but will not be much of a hassle to integrate.
 
-This is most likely the easiest way, but the less sustainable if you have lots of already defined smells in your own linter.
+This is most likely the easiest way, but the less sustainable if the `Paprika` model does not fit all your needs.
 
 **TODO: Add links & details**
 
 ### Add smells in the SmellDetector database
 
 If your smell detection are performed before persisting the model and already handled by a visitor pattern,
-you may want to add this detection before persisting the model, since spoon is already working with visitor.
-It would then be possible to add new metadata in the `SmellDetector` database, to represent the detected smell.
+you may want to add this detection before persisting the model, since Spoon is already working with visitor.
+It would then be possible to add new metadata in the `SmellDetector` database, to represent the detected code smells.
 
 
-The point is that you will have less work to do to transform you smell definition,
-but you would still be required to write a new detection query to look after all the already detected smells.
+The point is that you will have less work to do to transform your code smell definition.-
+However, if you also want to keep our originally detected code smells you would still be required to write a new detection query for them.
```

Comment:
However, if you also want to keep our originally detected code smells you would still be required to write a new detection query for them.
I think my original statement was not clear enough, I was meaning something along the line:
```suggestion
However, if you want to track your new code smells you would be required to write a new detection query for them.
```